client/cypress/integration/InfoBase/cypress-snapshots.js

```diff
@@ -16,7 +16,11 @@ module.exports = {
     },
     "Glossary": {
       "Tested on index-eng.html#glossary": {
-        "Axe violations allow list": null
+        "Axe violations allow list": {
+          "color-contrast, serious": [
+            "<button class=\"btn btn-ib-primary floating-button floating-button--fixed\" tabindex=\"-1\" aria-label=\"Back to top\" style=\"top: auto;\">Back to top</button>"
```

Comment:
Forgot to mention this in our dev meeting, but can't seem to even locate the error with the AXE tools

Reply:
Hm, it does seem that the AXE engine running in cypress isn't always in sync with the version that runs in the browser plugin. I think the browser plugins auto-update, while the cypress plugin will only update when our package locks do. Aside from that, I also don't know if the maintainers keep them in sync or use the same default configuration.

Aside from that, there's some potential differences in the page as examined in CI. The main one is the dev link banner being inserted (seems unrelated here). Might also be some other difference on the page, such as if you've scrolled/interacted with the page before running AXE locally. 

Outside of that, we've not been using this setup very long, so it could be some other kind of gotcha I've not accounted for. Worth looking in to it a bit more, since we need to get more acquainted with it, learn what the gotchas might be.